src/js/components/validateData/ValidateDataFileComponent.jsx

```diff
@@ -442,7 +434,7 @@ export default class ValidateDataFileComponent extends React.Component {
                                         tabIndex={0}
                                         className={clickDownloadClass}
                                         onKeyDown={clickDownloadOnKeyDownHandler}
-                                        onClick={clickDownload}
```

Comment:
Here we have `clickDownload` which may be `null` at the end of all of this, affected by `canDownload` but we didn't change `clickedReport` at all so how are we covering for this variable?

Reply:
Updated the `clickedReport` function to check for `this.state.canDownload`